recipes/avro/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.7.7" %}
+
+package:
+  name: avro
```

Comment:
As this is reusing the original name for Python code, it should be renamed to `python-avro`.